testpmd/hooks/roles/apply-cnf-cert-config/defaults/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
 # app_ns could also be: "{{ tnf_config[0].namespace|default('example-cnf') }}"
-app_ns: "{{ dci_openshift_app_ns|default('example-cnf') }}"
+app_ns: "{{ cnf_namespace|default('example-cnf') }}"
```

Comment:
what if we define cnf_namespace at the beginning of example-cnf-config hooks to use example-cnf value directly, avolding default values? This applies to the other cases too